backend/processing_image/Makefile

```diff
@@ -9,11 +9,11 @@ bump_version:
 	expr `cat VERSION` + 1 > VERSION
 
 build:
-	mkdir -p code/chalice/api_server
-	cp -R ../corpora/dataset_processing code/
-	cp -R ../corpora/common code/
-	cp -R ../requirements.txt code/
-	cp -R ../chalice/api_server/requirements.txt code/chalice/api_server/requirements.txt
+	mkdir -p code/backend/chalice/api_server
+	cp -R ../corpora code/backend
+	cp -R ../../requirements.txt code/
+	cp -R ../chalice/api_server/requirements.txt code/backend/chalice/api_server/requirements.txt
```

Comment:
why do you need the api_server/requirements.txt?

Reply:
The main requirements.txt references it